ts/components/screens/ScreenContentHeader.tsx

```diff
@@ -22,6 +22,7 @@ type Props = Readonly<{
   subtitleLink?: JSX.Element;
   dark?: boolean;
   dynamicHeight?: Animated.AnimatedInterpolation;
+  rightComponent?: React.ReactElement;
```

Comment:
This may reflect the same pattern highlighted [above](https://github.com/pagopa/io-app/pull/3292/files#r686831824), although it feels less problematic here being it a top-level component. Up to you :-)